recipe/meta.yaml

```diff
@@ -116,11 +116,38 @@ outputs:
         # 2022/03/29 hmaarrfk
         # It seems that this test can cause segmentation faults on the CIs. 
         {% set tests_to_skip = tests_to_skip + " or test_write_video_with_audio" %}
-        # 2021/10/29 hmaarrfk
-        # CIs do not have enough resources to run the full suite of model tests
-        - rm -f test/test_models.py  # [unix]
-        - del /f test\test_models.py  # [win]
-        - pytest --verbose -k "not ({{ tests_to_skip }})" test/
+        # 2022/07 hmaarrfk really large memory tests. Fail on CIs
+        {% set tests_to_skip = tests_to_skip + " or test_memory_efficient_densenet" %}
+        {% set tests_to_skip = tests_to_skip + " or test_resnet_dilation" %}
+        {% set tests_to_skip = tests_to_skip + " or test_mobilenet_v2_residual_setting" %}
+        {% set tests_to_skip = tests_to_skip + " or test_mobilenet_norm_layer" %}
+        {% set tests_to_skip = tests_to_skip + " or test_inception_v3_eval" %}
+        {% set tests_to_skip = tests_to_skip + " or test_fasterrcnn_double" %}
+        {% set tests_to_skip = tests_to_skip + " or test_googlenet_eval" %}
+        {% set tests_to_skip = tests_to_skip + " or test_fasterrcnn_switch_devices" %}
+        {% set tests_to_skip = tests_to_skip + " or test_mobilenet_v2_residual_setting" %}
+        {% set tests_to_skip = tests_to_skip + " or test_vitc_models" %}
+        {% set tests_to_skip = tests_to_skip + " or test_classification_model" %}
+        {% set tests_to_skip = tests_to_skip + " or test_segmentation_model" %}
+        {% set tests_to_skip = tests_to_skip + " or test_detection_model" %}
+        {% set tests_to_skip = tests_to_skip + " or test_detection_model_validation" %}
+        {% set tests_to_skip = tests_to_skip + " or test_video_model" %}
+        {% set tests_to_skip = tests_to_skip + " or test_quantized_classification_model" %}
+        {% set tests_to_skip = tests_to_skip + " or test_detection_model_trainable_backbone_layers" %}
+        {% set tests_to_skip = tests_to_skip + " or test_raft" %}
+        {% set tests_to_skip = tests_to_skip + " or test_build_fx_feature_extractor" %}
+        # I'm really not sure why test_read_partial_video is failing, but it seems 
+        # to pass locally so maybe it is also limited by memory?
+        {% set tests_to_skip = tests_to_skip + " or test_read_partial_video" %}
+        {% set tests_to_skip = tests_to_skip + " or test_incorrect_anchors" %}
+        {% set tests_to_skip = tests_to_skip + " or TestVideoGPUDecoder" %}
+        {% set tests_to_skip = tests_to_skip + " or test_generalizedrcnn_transform_repr" %}
+        {% set tests_to_skip = tests_to_skip + " or Tester" %}
+        {% set tests_to_skip = tests_to_skip + " or TestModelsDetectionNegativeSamples" %}
+        {% set tests_to_skip = tests_to_skip + " or TestVideoReader" %}
+        {% set tests_to_skip = tests_to_skip + " or TestVideo" %}
```

Comment:
How did you come up with this list? The first & last one probably match quite a bunch...

Reply:
Now that we have a passing test suite, we could try to peel off certain of those again and then inspect the durations. WDYT? We're waiting for new pytorch builds anyway....

Reply:
Go for it. 

I messed up the rebate for pttorch build 1 so i had to create a new 2 realse just now for version 1.12.0

Reply:
Feel free to rebase and cleanup as needed if you are going to go through this. I don't think I will.